README.md

```diff
@@ -18,7 +18,10 @@ Important parameters for the benchmarks and/or library are set in the config.h f
 
 For running benchmarks use the bench.sh script file that is provided in the main directory of this source tree.
 
-Example usage: `./bench.sh FILE.c OPTION1=NUM1  OPTION2=NUM2 ...`
+Example usage: `./bench.sh FILE.run OPTION1=NUM1  OPTION2=NUM2 ...`
+
+To perform smoke test: `./run_all.sh`. This will quickly run all available benchmarks with default options
+and store the results  in the file `results.txt`.
 
 The following options are available:
```

Comment:
I already removed the SPARC from the README.

Reply:
OK, another minor change is to remove the sparc assembly file sparc.il, since there is no need for it.

Reply:
Done